k8s/cassandra/apptest/tester/tests/basic-suite.yaml

```diff
@@ -0,0 +1,52 @@
+actions:
+- name: 3 nodes deployed successfully
+  bashTest:
+    script: |-
+      set -x
+      function info {
+        >&2 echo "${@}"
+      }
+
+      function get_desired_number_of_replicas_in_sts {
+        kubectl get sts "${STS_NAME}" \
+          --namespace "${NAMESPACE}" \
+          --output jsonpath='{.spec.replicas}'
+      }
+
+      function get_current_number_of_replicas_in_sts {
+        kubectl get sts "${STS_NAME}" \
+          --namespace "${NAMESPACE}" \
+          --output jsonpath='{.status.readyReplicas}'
+      }
+
+      function wait_for_healthy_sts {
+        info "Waiting for equal desired and current number of replicas"
+        while [[ $(get_current_number_of_replicas_in_sts) -ne $(get_desired_number_of_replicas_in_sts) ]]; do
+          info "Sleeping 10 seconds before rechecking..."
+          sleep 10
+        done
+        info "Statefulset has equal current and desired number of replicas"
+      }
+      STS_NAME="${APP_INSTANCE_NAME}-cassandra"
+      wait_for_healthy_sts
+      kubectl get sts "${STS_NAME}" \
+        --namespace "${NAMESPACE}" \
+        --output jsonpath='{.status.readyReplicas}'
+    expect:
+      statusCode:
+        equals: 0
+      stdout:
+        equals: '3'
+      stderr:
```

Comment:
maybe just rm this--asserting `statusCode` of 0 is sufficient? I think it's OK to output diagnosing messages to stderr, which we're not doing right now but we can add in the future.

Reply:
Sure, makes sense. Removed